.github/workflows/build-publish-develop-pr.yml

```diff
@@ -83,12 +83,28 @@ jobs:
             goarch: arm64
             dist_name: linux_arm64_v8.0
     steps:
-      - name: Checkout repository
+      - name: Checkout chainlink repository
         uses: actions/checkout@v4.2.1
         with:
           ref: ${{ env.CHECKOUT_REF }}
           fetch-depth: 0
 
+      - name: Setup Github Token
+        id: token
+        uses: smartcontractkit/.github/actions/setup-github-token@ef78fa97bf3c77de6563db1175422703e9e6674f # setup-github-token@0.2.1
+        with:
+          aws-role-arn: ${{ secrets.AWS_OIDC_GLOBAL_READ_ONLY_TOKEN_ISSUER_ROLE_ARN }}
+          aws-lambda-url: ${{ secrets.AWS_INFRA_RELENG_TOKEN_ISSUER_LAMBDA_URL }}
+          aws-region: ${{ secrets.AWS_REGION }}
+          set-git-config: "true"
```

Comment:
```suggestion
```

I think we can remove this since you're using the token explicitly in the next step?